README.md

```diff
@@ -24,6 +24,18 @@ Used API [Yandex.Speller](https://tech.yandex.ru/speller/doc/dg/concepts/About-d
 ## Using CLI
 `yaspeller [options] <file-or-directory-or-link...>`
 
+## Using with pre-commit:
```

Comment:
I would remove the `:` here -- you probably also want to link to pre-commit.com so it's more clear what this is referring to (and so others can find the appropriate documentation)

Reply:
Done, thanks!